src/types/Error.ts

```diff
@@ -0,0 +1,4 @@
+export type Error = {
```

Comment:
The type name 'Error' may cause confusion with the built-in Error class. Consider renaming it to something more specific, like 'TodoError'.